packages/realm/binding/react-native/NativeRealm.ts

```diff
@@ -0,0 +1,26 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2024 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { TurboModule, TurboModuleRegistry } from "react-native";
+
+export interface Spec extends TurboModule {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  getBinding(): Object;
+}
+
+export default TurboModuleRegistry.get<Spec>("Realm");
```

Comment:
I've seen both [TurboModuleRegistry.get()](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/turbo-modules.md#2-javascript-specification) and [TurboModuleRegistry.getEnforcing()](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/turbo-modules-xplat.md#1-javascript-specification) used but unsure if one is older or if there's even a difference.